content/articles/toukei_2kyu.md

```diff
@@ -0,0 +1,102 @@
+合格するために使用した書籍や勉強方法を残します。
+
+## 統計検定2級とは
+[統計検定２級｜統計検定：Japan Statistical Society Certificate](https://www.toukei-kentei.jp/exam/grade2/)
+
+## 筆者の経歴
+- 工業高校卒
+- 大学 工学部(情報工学)卒
+- ソフトウェアエンジニア(統計関連の業務を行ったことはない)  
+
+## 勉強開始前の知識
+- 大学で統計学(基礎レベル)の授業は受けていた。が、ほとんど忘れており、かろうじて確率分布や仮説検定の用語は覚えているぐらい。
+- 高校数学は得意ではない。
+
+## 使用した書籍およびサイト
+使用した書籍は以下の2冊です。
+- [日本統計学会公式認定　統計検定　2級　公式問題集［CBT対応版］](https://books.jitsumu.co.jp/book/b614221.html)
+- [日本統計学会公式認定　統計検定　2級　公式問題集［2018〜2021年］](https://books.jitsumu.co.jp/book/b590884.html)
+
+上記の2冊は問題集なので、以下のサイトを参考書代わりにしました。
+- [統計学の時間 | 統計WEB](https://bellcurve.jp/statistics/course/)
+  - Step1が2級の範囲になります。
+
+また、とけたろうさんのyoutubeチャンネルの動画も見ていました。動画は文字に比べると学習効率は下がりますが、お風呂などで流し見する分にはちょうどよかったです。動画もかなりわかりやすいので、統計Webのサイトを見ても理解ができなかった単元などを確認する用途としても閲覧していました。
+- (とけたろうチャンネル)[https://www.youtube.com/channel/UCxxLbPmmSud_xHZi45SoOXQ]
+
+## 勉強期間と勉強時間
+- 勉強期間
+  - 6/1~7/20の約2ヶ月
+- 1日の勉強時間
+  - 1~2時間(7月に入ってからは休日5時間) 途中何日かはサボっています
+- 合計勉強時間
+  - 約100時間ほど  
+
+## 勉強方法
+まず、試験日までの2ヶ月間の勉強スケジュールを立てました。今回は2ヶ月と長い勉強時間を確保できたので、少し余裕があるスケジュールです。(特に6月)
+
+| 日付 | 行うこと |
+| -- | -- |
+| 6/1 | 過去問を1回解いてみる |
+| 6/2-6/30| 試験範囲の全体像と具体的な公式、用語の把握 & 問題集を11周目 |
+| 7/1-7/5 | 6月の学習でまったく頭に入っていない単元(公式が思い出せないレベル)を再度学習 |
+| 7/6-7/9 | 6月の学習でちょっと頭に入っていない単元(公式は思い出せるが、捻った問題が解けないレベル)を再度学習 |
+| 7/10-7/11 | 問題集 3周目 |
+| 7/12-7/19 | 問題集&過去問無限ループ |
+| 7/20 | 受験当日 |
+
+### 6/1 過去問を1回解いてみる 
+統計検定2級がどんなものかを知るために、公式ページにある2021年6月の問題を解いてみました。正答率 2-3割でした。
+※ 後から知りましたが、2021年6月の試験は傾向がかなり違うので、あまり参考にならないらしいです。
+
+### 6/2-6/30 試験範囲の全体像と具体的な公式、用語の把握 & 問題集を11周目
+1ヶ月かけて、以下を1周しました。
+- [統計学の時間 | 統計WEB](https://bellcurve.jp/statistics/course/)
+- [とけたろうチャンネル](https://www.youtube.com/channel/UCxxLbPmmSud_xHZi45SoOXQ)
+- [日本統計学会公式認定　統計検定　2級　公式問題集［CBT対応版］](https://books.jitsumu.co.jp/book/b614221.html)
+
+この時点では、完璧に覚えるのは目指しません。あくまで全体像の把握と公式、用語をさらっと知ることを目指します。
+
+### 7/1-7/5 6月の学習でほとんど理解していない単元(公式が思い出せないレベル)を再度学習
+統計検定2級の出題範囲を確認し、各単元を以下のように分けました。
+1. ほぼ理解している
+2. ちょっと理解していない
+3. ほとんど理解していない
+
+そこから、2と3に絞って、2回目の学習を開始します。
+
+### 7/6-7/9 6月の学習でちょっと理解していない単元(公式は思い出せるが、捻った問題が解けないレベル)を再度学習
+同上。
+
+### 7/10-7/11 問題集 3周目
+7月に入ってからの学習状況を確かめるために、問題集をもう1回解き直しました。ここまでくると、答えを覚えているのもありますが、問題集の問題はほぼ解ける状態になっていました。
+
+### 7/12-7/19 問題集&過去問無限ループ
+この時点で公式や用語はほぼ頭に入っており、「今日受けても多分受かりそう」くらいになっていました。唯一の懸念は高校数学の確率ですが、試験に出るのは1-2問くらいだろうと思い、諦めました。
+
+初見で解いた過去問も8-9割だったので、自信はついていました。
+
+### 7/20 受験当日
+受験は18:00開始だったので、当日も4時間ほど勉強しました。朝から勉強すれば、7時間以上の勉強時間を確保できますが、試験の集中力が切れるのを懸念して、極力やすみながら勉強をしていました。
+
+## 結果
+69/100 で合格でした。過去問を初見で解いた時は、8割以上とれることが多かったので、優秀賞とかもらえるかなと思っていたのですが、そんなに甘くなかったです。
+
+## 感想
+
+思ったより難しかったなと思いました。過去問を初見で解いた時は8-9割は取れていたので、かなり自信を持っていたのですが、本番の問題が想像以上に難しかったです。正答していると自信を持って言える問題が少なく、不合格が頭をよぎりましたね。
+また、90分という試験時間もかなりギリギリで、最後の問題を解いた時点で残り1分だったので見直しもできなかったです。(マークミスをしていないかのチェックくらいしかできませんでした)
+
+過去問だとただ公式を当てはめるだけみたいな問題もありましたが、CBTだとほとんどありませんでした。なので、公式などの意味を理解せずただ公式や過去問の答えを丸暗記するような勉強方法だと受からないような気がします。
+
+不満と言えば...CBTは試験終了後すぐに受験したPC上で合否が分かるようになっていますが、せめて合否を出す前になにか表示して欲しかったですね...アンケート入力後、次へボタンを押したらすぐさま結果が出てきたのでちょっとびっくりしました。
```

Comment:
🚫 **[textlint]** <eslint.rules.ja-technical-writing/sentence-length> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line 99 sentence length(119) exceeds the maximum sentence length of 100.
Over 19 characters. (ja-technical-writing/sentence-length)
<!-- __reviewdog__:ChBhNjhhYTE4YjdjOTU1YmE1Egh0ZXh0bGludA== -->